.github/workflows/hypertrace-ingester/docker-compose.yml

```diff
@@ -82,6 +82,8 @@ services:
       default:
         aliases:
           - pinot-controller
+          - pinot-server
```

Comment:
@laxmanchekka This has fixed the e2e test for hypetrace-ingester based on docker-compose. Recently, I have added new alias and so, it was failing.